dhcpv6/option_nontemporaryaddress_test.go

```diff
@@ -39,6 +39,15 @@ func TestOptIANAParseOptIANAInvalidOptions(t *testing.T) {
 	require.Error(t, err)
 }
 
+func TestOptIANADelOption(t *testing.T) {
```

Comment:
You should also test the case of multiple options, subsequent and non-subsequent